ClinGen/clincoded
# Connects to #1368. Additional upgrade scripts.

**Technical notes:**
1. Updated the `computational.json`, `evaluation.json` and 'curatorHistory.json` schemas.
2. Updated the `computational.json`, `evaluation.json` test data.
3. Added new users to the test `user.json` file.
4. Created new db dumps.
5. Uncommented the `batchupgrade` command in the `cloud-config.yml` file that was previously commented out for testing purpose.
6. Fixed error in the `gdm.py` upgrade script.
7. Fixed error and added a condition in the `interpretation.py` upgrade script.
8. Added upgrade_step for migrating diseases in `computational.py` upgrade script.
9. Created `evaluation.py` and `history.py` upgrade scripts to migrate associated diseases.

**Prerequisites:**
The new MonDO-based disease records had been loaded into the test/review instance. As of today, there are 70 unique orphaPhenotype disease to be migrated to MonDO-based diseases.

**Steps to test:**
1. Go to `^/users/` and expect to see the `Lisa Kurtz` user on the list.
2. Go to `^/search/` and compare the main item types against those of production. Expect to see the counts of individual item types should be matching.
3. Go to `https://r11-rc1-ebe540c-jzhen.demo.clinicalgenome.org/histories/a8c02e2a-eaf4-4fe3-9ac0-c6d7bac5da66/` and expect to see `meta.gdm.disease` showing the MonDO-based disease info.
